api/graphql/schema.graphql

```diff
@@ -165,10 +162,189 @@ input UpdateUserInput {
     preferredLang: Language @rename(attribute: "preferred_lang")
 }
 
+input LocalizedStringInput {
+    en: String
+    fr: String
+}
+
+input CreateUserBelongsTo {
+    connect: ID
+}
+
+input CreatePoolBelongsTo {
+    connect: ID
+}
+
+input CreateClassificationInput {
+    name: LocalizedStringInput
+    group: String!
+    level: Int!
+    minSalary: Int @rename(attribute: "min_salary")
+    maxSalary: Int @rename(attribute: "max_salary")
+}
+
+input CreateClassificationBelongsToMany {
+    create: [CreateClassificationInput!]
+    connect: [ID!]
+}
+
+input CreateCmoAssetInput {
+    key: String!
+    name: LocalizedStringInput!
+    description: LocalizedStringInput
+}
+
+input CreateCmoAssetBelongsToMany {
```

Comment:
I think sync is the most appropriate operation for the CmoAssetBelongsToMany, OperationalRequirementBelongsToMany, and ClassificationBelongsToMany. It would be the easiest way to process a multi-select form input.